app/components/embed/pdf_component.html.erb

```diff
@@ -1,10 +1,13 @@
 <%= render Embed::CompanionWindowsComponent.new(viewer:, stimulus_controller: 'file-auth') do |component| %>
   <% component.with_header_button do %>
+    <%# chromium has a bug preventing the esc key from exiting full screen in some situations %>
```

Comment:
would you mind linking to https://issues.chromium.org/issues/40131988 here.

Reply:
good call, will do